repl/src/test/scala/org/apache/spark/repl/ReplSuite.scala

```diff
@@ -386,8 +386,7 @@ class ReplSuite extends SparkFunSuite with BeforeAndAfterAll {
     // Ensure log level threshold for REPL is ERROR.
     assertContains(replLoggerLogMessage + "ERROR", out)
 
-    assertDoesNotContain(debugLogMessage1, out)
-    assertContains(warnLogMessage1, out)
+    assertDoesNotContain(warnLogMessage1, out)
```

Comment:
This is what it is before the migration. For the logger inherited root logger config (i.e. without custom configuration), this is same behavior as before.